src/api/oAuthApi.ts

```diff
@@ -0,0 +1,10 @@
+import { AxiosResponse } from 'axios';
+import api from './api';
+
+export const getServiceId = (payload: string): Promise<AxiosResponse> => (
+  api.get(`/oauth/yandex/service-id?redirect_uri=${payload}`)
+);
+
+export const signinYandex = (payload: Record<string, string>): Promise<AxiosResponse> => (
```

Comment:
❗️
По-хорошему в кэмел кейсе бьётся каждое слово)
```suggestion
export const signInYandex = (payload: Record<string, string>): Promise<AxiosResponse> => (
```

Reply:
Поправила, но мы как-то с самого начала использовали signin и signup - как одно слово. Signin - я поправила везде, а signup - не стала, так как там много править - название файла для страницы, схему и т.д. Не уверена, что оно того стоит. Но если будет желание, можно будет поправить